Draft v1.0/Specification.md

```diff
@@ -1,6 +1,8 @@
+<img src="https://openpreservation.org/wp-content/uploads/2019/11/opf-site-logo.png" width="300" alt="Open Preservation Foundation logo">
```

Comment:
Use the markdown syntax for this please and chose an image of the right size, although you can add classes to GH markdown if memory serves.

Reply:
If you don't want HTML to conrtrol the width, how do you propose we host an image file with the exact width?